lychee-bin/src/formatters/stats/compact.rs

```diff
@@ -51,25 +53,40 @@ impl Display for CompactResponseStats {
             writeln!(f)?;
         }
 
-        color!(f, NORMAL, "\u{1F50D} {} Total", stats.total)?;
+        color!(f, NORMAL, "🔍 {} Total", stats.total)?;
```

Comment:
@mre I'd personally prefer this inlined version as it is much more readable compared to the code and a comment at the end of the line (if any). Was there a reason you don't use the emoji directly?

Reply:
Talked to @m3t0r and we both don't see a reason why using the emoji directly is an issue. So, sure, let's change it back. Easier to read anyway.

Reply:
Sounds great :+1: